index.toml

```diff
@@ -4,29 +4,31 @@ toc = true
 colab = "https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/"
 
 [[tutorial]]
-title = "Build Your First QA System"
+title = "Build Your First Question Answering System"
 description = "Get Started by creating a Retriever Reader pipeline."
 level = "beginner"
 weight = 10
-notebook = "01_Basic_QA_Pipeline.ipynb"
+notebook = "01_build_your_first_question_answering_system.ipynb"
 aliases = ["first-qa-system"]
-slug = "01_Basic_QA_Pipeline"
+slug = "01_basic_qa_pipeline"
 
 [[tutorial]]
-title = "Fine-Tuning a Model on Your Own Data"
+title = "Fine-Tune a Reader"
 description = "Improve the performance of your Reader by performing fine-tuning."
 level = "intermediate"
 weight = 50
-notebook = "02_Finetune_a_model_on_your_data.ipynb"
-aliases = ["fine-tuning-a-model"]
+notebook = "02_finetune_a_reader.ipynb"
+aliases = ["fine-tuning-a-model", "02_finetune_a_model_on_your_data", "fine-tune-a-model"]
```

Comment:
No need for "02_finetune_a_model_on_your_data", "fine-tune-a-model" here.
aliases aren't great, they do a weird redirect. So we should only use them to maintain legacy. 
"02_finetune_a_model_on_your_data" --> already the slug so no need for the alias